Explorer/Assets/DCL/SDKComponents/AvatarAttach/Systems/AvatarAttachHandlerSetupSystem.cs

```diff
@@ -21,11 +24,14 @@ namespace DCL.SDKComponents.AvatarAttach.Systems
     [LogCategory(ReportCategory.AVATAR_ATTACH)]
     public partial class AvatarAttachHandlerSetupSystem : BaseUnityLoopSystem
     {
+        private static readonly QueryDescription AVATAR_QUERY = new QueryDescription().WithAll<AvatarBase, AvatarShapeComponent>();
```

Comment:
Not used

Reply:
Tackled at https://github.com/decentraland/unity-explorer/pull/3430/commits/8861ff81ea16779ee379fb547fd0353f842945fc